react/Viewer/ImageViewer.spec.jsx

```diff
@@ -32,12 +33,14 @@ const setup = () => {
   const root = render(
     <DemoProvider>
       <BreakpointsProvider>
-        <ImageViewer
-          file={file}
-          gestures={gestures}
-          gesturesRef={{}}
-          onSwipe={jest.fn()}
-        />
+        <EncryptedProvider url={''}>
```

Comment:
if you disagree with JF, at least:
```suggestion
        <EncryptedProvider url="">
```